### DISCUSSION TITLE
Two routes to the same component?

### DISCUSSION THREAD

**Discussion by Josrph** [vercel/next.js#44119] [Help] [answered]
I have two different routes that point to the same component:

```
/post/pageSize/pageIdx => <Posts />
/postType/postTypeId/post/pageSize/pageIdx  => <Posts />
```
Here's what the paths look like when the dynamic parts are populated:

```
/post/1/1
/category/1/post/1/1
```

Is it possible in Next.js and How?

**Comment by icyJoseph** [comment, marked as answer]
In next.js routes are based from the file system.

So yes it is possible by creating:

- A file at `pages/posts/[pageSize]/[pageIdx].tsx` 
- A file at `pages/[postType]/[postTypeId]/post/[pageSize]/[pageIdx].tsx`

And just export how you want that page to look like.
